projects/1.18/assets/pyromancer/pyromancer/lang/zh_cn.json

```
@@ -1,226 +1,299 @@
+  "effect.pyromancer.volcanic_mark_effect": "火山印记",
+  "effect.volcanic_mark_effect": "火山印记",
+  "enchantment.pyromancer.ashen_pages.desc": "烈焰日志不再能在炽焰台处充能，而是在使用其击杀目标时充能。",
   "enchantment.pyromancer.ashen_pages": "书页付灰",
-  "enchantment.pyromancer.ashen_pages.description.temp11wuzg": "（以下描述翻译自 CurseForge 页面；烈焰日志的附魔不再重复翻译）在炽焰台上手动填充烈焰粉不再可行，但烈焰会随时间推移而缓慢恢复",
+  "enchantment.pyromancer.blazing_wounds.desc": "斧的攻击能让着火的目标体内迸溅出岩浆。",
   "enchantment.pyromancer.blazing_wounds": "§6烈焰创伤",
-  "enchantment.pyromancer.breakthrough": "突破",
-  "enchantment.pyromancer.breakthrough.description.temp11wuzg": "钉锤：攻击使盾进入冷却",
+  " enchantment.pyromancer.breakthrough.desc": "钉锤的攻击能禁用盾牌。与“压制”冲突。",
+  "enchantment.pyromancer.breakthrough": "突进",
+  "enchantment.pyromancer.crowd_control.desc": "疣猪鞭对群体目标造成更多伤害。",
+  "enchantment.pyromancer.crowd_control": "群控",
+  "enchantment.pyromancer.elongation.desc": "疣猪鞭的攻击范围增加。",
+  "enchantment.pyromancer.elongation": "延长",
+  "enchantment.pyromancer.goliath_slayer.desc": "弹弓的弹射物对大型目标造成更多伤害。",
   "enchantment.pyromancer.goliath_slayer": "歌利亚杀手",
-  "enchantment.pyromancer.goliath_slayer.description.temp11wuzg": "弹弓：对更大的敌人造成更高的伤害",
-  "enchantment.pyromancer.hook": "钩引",
-  "enchantment.pyromancer.hook.description.temp11wuzg": "疣猪鞭：把你引向钩住的敌人",
-  "enchantment.pyromancer.infernal_harvest": "§6地狱丰收",
+  "enchantment.pyromancer.infernal_harvest.desc": "锄的攻击能点燃较大范围内的目标。",
+  "enchantment.pyromancer.infernal_harvest": "§6炼狱丰收",
+  "enchantment.pyromancer.legerdemain.desc": "弹弓有机会重复发射当前同种弹射物。",
   "enchantment.pyromancer.legerdemain": "戏法",
-  "enchantment.pyromancer.legerdemain.description.temp11wuzg": "弹弓：弹射物有几率复制",
-  "enchantment.pyromancer.lightspeed": "光速",
-  "enchantment.pyromancer.lightspeed.description.temp11wuzg": "疣猪鞭：攻击几乎是即时的，但伤害相对更低",
+  "enchantment.pyromancer.resilence.desc": "弹弓弹射物的飞行距离增加。",
   "enchantment.pyromancer.resilience": "弹性",
-  "enchantment.pyromancer.resilience.description.temp11wuzg": "弹弓：弹射物距离更远",
+  "enchantment.pyromancer.rush.desc": "如果你的移动速度快于目标，则钉锤能造成更多伤害。",
   "enchantment.pyromancer.rush": "冲刺",
-  "enchantment.pyromancer.rush.description.temp11wuzg": "钉锤：如果你比敌人速度更大，则造成更高的伤害",
+  "enchantment.pyromancer.scalding_spikes.desc": "钉锤的攻击能让目标体内爆发出火舌。",
   "enchantment.pyromancer.scalding_spikes": "§6炽热尖刺",
-  "enchantment.pyromancer.sturdiness": "强力",
-  "enchantment.pyromancer.sturdiness.description.temp11wuzg": "对护甲值越高的敌人伤害越高，伤害加成最大为 5 点",
-  "enchantment.pyromancer.tamer": "驯兽师",
-  "enchantment.pyromancer.tamer.description.temp11wuzg": "疣猪鞭：使周围的疣猪兽昏迷而非惊惶逃走",
-  "enchantment.pyromancer.two_handed": "双持",
-  "enchantment.pyromancer.two_handed.description.temp11wuzg": "钉锤：副手空闲时伤害更高且有几率使敌人昏迷",
+  "enchantment.pyromancer.sturdiness.desc": "钉锤会无视目标的部分护甲值。",
+  "enchantment.pyromancer.sturdiness": "坚固",
+  "enchantment.pyromancer.suppression.desc": "钉锤的攻击能穿透盾牌。与“突进”冲突。",
+  "enchantment.pyromancer.suppression": "压制",
+  "enchantment.pyromancer.traction.desc": "疣猪鞭能以更大力将目标拖拽过来。",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "enchantment.pyromancer.traction.desc": "疣猪鞭能以更大的力量拖拽目标。",
```

原文：**Hogwhip pulls enemies with greater strength **